app/stac_api/utils.py

```diff
@@ -395,7 +395,8 @@ def get_dynamic_max_age_value(update_interval):
     '''Get the max_age value for dynamic cache based on `update_interval` DB field
 
     -       update_interval < 0  then use default cache settings
-    -  0 <= update_interval < 10 then no cache
+    -  0 == update_interval then no cache
+    -  0 < update_interval < 10 then cache of 1s
```

Comment:
maybe a comment here about the relation to compression behaviour of CF could be useful to understand why this was done that way in the future.